### PR TITLE
keyword_case upper

### LINKED ISSUE #941: lint rule for checking pretty print (keyword upper case)
We'd better enforce the keyword upper case as discussed in #780, #893, and [here about problem with `CLASS ZCL_<ALLUPPERCASE> IMPLEMENTATION` when using the Class Builder SE24/SE80](https://github.com/abap2xlsx/abap2xlsx/pull/937#issuecomment-1003525533) (whatever using the form-based or source code-based editor, it's changed to upper case after activation).

Concerned abap lint rule:
https://github.com/abap2xlsx/abap2xlsx/blob/master/abaplint.json#L109

@pokrakam @AndreaBorgia-Abo @larshp What do you think?

### PR DESCRIPTION
**Pull request by sandraros**
Fix #941

### CODE REVIEW

**Comment by larshp** [comment]
`CP` is a false positive, I'm on it... https://github.com/abaplint/abaplint/pull/2324

UPDATE: fixed

**Review by larshp — APPROVED** [review]

File: src/zcl_excel_style_changer.clas.abap [hunk 1]
```diff
@@ -63,7 +63,7 @@ ENDCLASS.
 
 
 
-CLASS zcl_excel_style_changer IMPLEMENTATION.
+CLASS ZCL_EXCEL_STYLE_CHANGER IMPLEMENTATION.
```
Comment (AndreaBorgia-Abo) [inline src/zcl_excel_style_changer.clas.abap:66]: I'm intrigued that this minor change here triggered a flurry of moves below!

File: src/zcl_excel_style_changer.clas.abap [hunk 2]
```diff
@@ -187,9 +187,9 @@ CLASS zcl_excel_style_changer IMPLEMENTATION.
       stylemapping-complete_stylex-borders-right = complete_stylex-borders-right.
     ENDIF.
 
-    IF multiple_change_requested-borders-TOP = abap_true.
-      stylemapping-complete_style-borders-TOP = complete_style-borders-TOP.
-      stylemapping-complete_stylex-borders-TOP = complete_stylex-borders-TOP.
+    IF multiple_change_requested-borders-top = abap_true.
```
Comment (AndreaBorgia-Abo) [inline src/zcl_excel_style_changer.clas.abap:190]: This one too is welcome.

File: src/zcl_excel_style_changer.clas.abap [hunk 3]
```diff
@@ -536,1140 +536,1138 @@ CLASS zcl_excel_style_changer IMPLEMENTATION.
...
 
   ENDMETHOD.
 
 
-  METHOD zif_excel_style_changer~set_fill_gradtype_position2.
+  METHOD zif_excel_style_changer~set_complete_borders_all.
 
-    complete_style-fill-gradtype-position2 = value.
-    complete_stylex-fill-gradtype-position2 = 'X'.
+*    lv_border_supplied = boolc( ip_borders_allborders IS SUPPLIED ).
```
Comment (AndreaBorgia-Abo) [inline src/zcl_excel_style_changer.clas.abap]:
```suggestion

```

**Review by AndreaBorgia-Abo — CHANGES_REQUESTED** [review]
There's a commented-out line that might be worth removing and a large number of code lines which got shuffled around but otherwise it seems ok.

**Comment by larshp** [comment]
SE80/SE24 and sometimes Eclipse reorders the methods, its annoying, but this seems like a good change to have it in. There are no logical changes in this PR, so its easier to check the reordering.

File: src/zcl_excel_style_changer.clas.abap (hunk 1 above)
Comment (sandraros) [reply src/zcl_excel_style_changer.clas.abap:66]: As the issue was about the Class Builder, I used it and a few methods have been reordered, and it messed up the diff! 😀

**Pull request merged**